controllers/sms-gateway.js

```diff
@@ -73,9 +80,11 @@ function getOutgoing(callback) {
         content: message.message,
       };
     });
-    markMessagesForwarded(messages, function() {
+    // TODO add this back in once this function also queries for
+    // messages in state: 'forwarded-to-gateway'
+    // markMessagesForwarded(messages, function() {
```

Comment:
That would require two db requests or a view change. I recommend the view change but not for 0.4.